build.psm1

```diff
@@ -313,7 +313,7 @@ function Start-PSBuild {
     }
 
     if ($Clean) {
-        Write-Log "Cleaning your working directory. You can also do it with 'git clean -fdX --exclude .vs/PowerShell/v16/Server/sqlite3'"
+        Write-Log -message "Cleaning your working directory. You can also do it with 'git clean -fdX --exclude .vs/PowerShell/v16/Server/sqlite3'"
```

Comment:
@JamesWTruher Could you please elaborate a bit why `-message` needs to be specified?
It is a positional parameter (`position = 0`), so it looks to me should have worked as expected.
```powershell
PS:4> gcm Write-Log | % Definition

    param
    (
        [Parameter(Position=0, Mandatory)]
        [ValidateNotNullOrEmpty()]
        [string] $message,

        [switch] $isError
    )
    if ($isError)
    {
        Write-Host -Foreground Red $message
    }
    else
    {
        Write-Host -Foreground Green $message
    }
    #reset colors for older package to at return to default after error message on a compilation error
    [console]::ResetColor()
```

Reply:
Generally, positional parameters are less preferred than being explicit (we have an analyzer rule for this). While I was making the changes elsewhere to the use of `Write-Log`, I thought I should do it here.